.travis.yml

```diff
@@ -136,29 +105,23 @@ jobs:
 
     - name: "Run redeployment integration tests"
       stage: redeployment integration tests
-      if: commit_message !~ \[skip-integration-tests\] OR branch = master
```

Comment:
Why remove this condition?

Reply:
From talking to @laurenfrederick, the conditional that determines if redeployment tests are run is here: https://github.com/nasa/cumulus/blob/9bb2dd453a3b07c37e638899c9f3f21ec2358501/.travis.yml#L11, so this conditional was not needed.